docs/enterprise/region-balancer.md

```diff
@@ -0,0 +1,34 @@
+# Region Balancer
+
+This plugin balances the write load of regions across datanodes, using specified window sizes and load thresholds to prevent frequent region migrations.
+
+```toml
+[[plugins]]
+[plugins.region_balancer]
+
+max_window = "45s"
+
+window_stability_threshold = 2
+
+min_load_threshold = 10485760
```

Comment:
We should let human can read it.

Reply:
I see. Let me update the plugin options.